src/test/java/com/google/api/codegen/testdata/discoveries/py/py_translate.v2.json.baseline

```diff
@@ -12,16 +12,12 @@ from pprint import pprint
 
 from googleapiclient import discovery
 
+developerKey = ''  # TODO: Update placeholder with desired API key.
 
-# TODO: Change placeholder below to desired API key:
-service = discovery.build('translate', 'v2', developerKey='{MY-API-KEY}')
-
-
-# TODO: Change placeholder below to desired parameter value for the `list` method:
+service = discovery.build('translate', 'v2', developerKey=developerKey)
 
 # The text to detect
-q = ''
-
+q = []  # TODO: Update placeholder value.
```

Comment:
We generated a string before, but I believe we were wrong, I tracked down the documentation and example from `google-api-python-client`:


Documentation:
`q: string, The text to detect (required) (repeated)`

Sample:
https://github.com/google/google-api-python-client/blob/master/samples/translate/main.py#L39
```
print(service.translations().list(
    source='en',
    target='fr',
    q=['flower', 'car']
).execute())
```